lib/stripe/api_operations/update.rb

```diff
@@ -1,57 +1,9 @@
 module Stripe
   module APIOperations
     module Update
```

Comment:
Thoughts on just making `.update` available everywhere that `#save` is? I can't think of any particular reason not to expose this functionality given that it seems to operate from the same endpoint.